media/css/firefox/whatsnew/whatsnew-98-turningred-en.scss

```diff
@@ -0,0 +1,179 @@
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import 'includes/base';
+
+$notebook-green: #9ff0d7;
+$notebook-green-light: #cbf3e6;
+$notebook-red: rgba(246, 147, 169, 0.5);
+$notebook-gradient: #fff radial-gradient(circle at bottom, $notebook-green 1px, transparent 0) top/ 1px 36px;
+$notebook-gradient-light: #fff radial-gradient(circle at bottom, $notebook-green-light 1px, transparent 0) top/ 1px 36px;
+
+.content-wrapper {
+    background: $notebook-gradient;
+    position: relative;
+
+    // note book red line
+    &::before {
+        background: $notebook-red;
+        content: '';
+        height: 100%;
+        left: 10%;
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        width: 1px;
+        z-index: 0;
+    }
+}
+
+.c-page-header {
+    .mzp-l-content {
+        padding-top: $spacing-md;
+        padding-bottom: $spacing-md;
+    }
+}
+
+// * -------------------------------------------------------------------------- */
+// Main content
+.wnp-content-main {
+    padding: 0;
+    text-align: center;
+    color: $color-black;
+
+    .mzp-l-content {
+        padding-top: 0;
+    }
+}
+
+.wnp-main-image {
+    margin: $layout-sm auto;
+}
+
+.wnp-main-title {
+    @include text-title-lg;
+    color: $color-black;
+    max-width: $content-md;
+    margin-left: auto;
+    margin-right: auto;
+
+    strong {
+        font-weight: inherit;
+        position: relative;
+
+        &::after {
+            background: transparent url('/media/img/firefox/whatsnew/whatsnew98/underline-red.svg') center center no-repeat;
+            background-size: contain;
+            content: '';
+            display: block;
+            height: 0.4em;
+            left: -0.1em;
+            position: absolute;
+            top: calc(100% - 0.2em);
+            width: calc(100% + 0.2em);
+        }
+    }
+}
+
+.wnp-main-tagline {
+    @include text-body-xl;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: $content-md;
+}
+
+.wnp-main-cta {
+    margin: $layout-md auto;
+}
+
+.wnp-secondary-cta {
+    position: relative;
+    z-index: 10;
+}
+
+.c-common-things {
+    color: $color-black;
+    text-align: center;
+
+    .mzp-l-content {
+        padding-bottom: 0;
+        padding-top: 0;
+    }
+
+    h2 b {
+        font-weight: inherit;
+        position: relative;
+
+        &::after {
+            background: transparent url('/media/img/firefox/whatsnew/whatsnew98/circle-red.svg') center center no-repeat;
+            background-size: contain;
+            content: '';
+            display: block;
+            height: calc(100% + 0.7em);
+            left: -0.3em;
+            position: absolute;
+            top: -0.35em;
+            width: calc(100% + 1em);
+        }
+    }
+}
+
+// Sticky gradient only shown if JS is available
+.js .c-common-things {
+    padding-top: 8em; // this plus the negative top margin pushes the gradient down... hacky, I know.
```

Comment:
**question (non-blocking):** Not sure I follow why the hack is needed? If I remove both padding-top and margin-top in dev tools, the bottom fade behaviour seems the same.

Reply:
I don't want the gradient to cover the headline, I wanted it to stop just short of the top of the list. The padding makes room for the headline and the negative margin compensates for the padding. I knew it was hacky, hence the comment. There may be a cleaner way to do it, maybe relative positioning...

Reply:
And actually looking at it again I think I want even a little more space... bumping to 10em.

Reply:
Ah, ok, I see it now. When I remove those spacing values, the header has the gradient. 
<img width="576" alt="Screen Shot 2022-03-03 at 4 35 56 PM" src="https://user-images.githubusercontent.com/19650432/156609283-235946a6-cb2b-4b24-a565-c050f84dc177.png">

Small hack on a short-lived page doesn't bother me. Does the job 👍